Source/Core/Common/AndroidAnalytics.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+#pragma once
+
+#include <string>
+
+#include "Common/Analytics.h"
+
+namespace Common
+{
+typedef void (*SendReport)(std::string, std::string);
```

Comment:
Any particular reason this is a `typedef` instead of a `std::function` (like, `using SendReport = std::function<void(std::string, std::string)>`)?

Reply:
Didn't realize std::function was a thing. I'll change that.